hadoop-ozone/dev-support/intellij/ozone-site.xml

```diff
@@ -71,4 +71,24 @@
     <name>datanode.replication.port</name>
     <value>0</value>
   </property>
+  <property>
+    <name>ozone.scm.ratis.enable</name>
+    <value>false</value>
+  </property>
+  <property>
+    <name>hdds.container.report.interval</name>
+    <value>60s</value>
+  </property>
+  <property>
```

Comment:
we can have the config with default value and respective dev-config can be changed as required while test.

Reply:
done